itest/lib/createTestBrim.js

```diff
@@ -36,6 +38,25 @@ export default (name: string) => {
       )
     },
 
+    setValue(locator: Locator, value: string) {
+      return appStep(`set input ${locator.css} to ${value}`, () =>
+        app.client
+          .waitForVisible(locator.css)
+          .then(() => app.client.setValue(locator.css, value))
+      )
+    },
```

Comment:
This is an example of a great function for a test to use because the wait-until-I-can-do-something-then-do-it dance is handled here, not requiring the test to do anything.

Reply:
Aww yes, I see what you mean. Yeah, it's nice to have that type of code in here.